frontend/components/EditItem.js

```diff
@@ -1,21 +1,40 @@
 import React from "react";
 import { useForm } from "react-hook-form";
-import { mutate } from "swr";
+//import { mutate } from "swr";
 import InputText from "./InputText";
 import Button from "./Button";
-import { fetchIt } from "../utils";
+import { adminFetchIt, AUTH0_API_IDENTIFIER } from "../utils";
+import PropTypes from "prop-types";
+import { useAuth } from "use-auth0-hooks";
+import { registryType } from "../types";
+import Link from "./Link";
 
-const EditItem = () => {
-  const { register, handleSubmit, errors, reset } = useForm();
+const audience = AUTH0_API_IDENTIFIER;
+
+const EditItem = ({ registry }) => {
+  const { _id } = registry;
+  const registryId = _id;
+  const { accessToken } = useAuth({ audience });
+  const { register, handleSubmit, errors /*,reset*/ } = useForm();
   const onSubmit = async formData => {
-    mutate("/items", async items => {
-      const newItem = await fetchIt("/item", {
-        method: "POST",
-        body: JSON.stringify(formData),
-      });
-      return [...items, newItem];
+    await adminFetchIt(`/item/registry/${registryId}`, accessToken, {
```

Comment:
Yup, that's the correct fetch. It should be wrapped with a `try... catch` to catch any errors though.